src/runtime/server/server.ts

```diff
@@ -88,6 +89,7 @@ export function create(appState: AppState) {
           path: settings.data.path,
           playgroundPath: settings.data.playground ? settings.data.playground.path : undefined,
         })
+        DevMode.sendServerReadySignalToDevModeMaster()
```

Comment:
A bit OT from the main PR but this was a regression from the server refactor, causing the `onAfterRestart` hook to never be triggered anymore. I'll probably merge-rebase the PR to keep the commit for that fix